lti_tool/templates/lti_tool/lti_submission.html

```diff
@@ -0,0 +1,20 @@
+{% extends "exercise/submission.html" %}
+
+{% block header %}{% endblock %}
+{% block course_sidebar %}
+<div class="col-sm-1"></div>
+{% endblock %}
+{% block breadcrumb %}{% endblock %}
+{% block breadcrumblist %}{% endblock %}
+{% block siblings %}{% endblock %}
+{% block siblings_bottom %}{% endblock %}
+{% block footer %}{% endblock %}
+
+{% block exercise_wait %}
+	{% include "lti_tool/_lti_exercise_wait.html" %}
+{% endblock %}
```

Comment:
The include shouldn't be indented.

Reply:
Why not? It is easier to read when the code inside a block or tag is indented more than the surrounding block/tag.